test/cxx/Core/ApplicationPool/PoolTest.cpp

```diff
@@ -2026,4 +2026,29 @@ namespace tut {
 
 
 	/*****************************/
+
+	TEST_METHOD(86) {
+		// If a request is in the getWaitlist for longer than maxRequestQueueTime,
+		// then an exception is returned.
+		Options options = createOptions();
+		options.appGroupName = "test1";
+                options.maxRequestQueueSize = 2;
```

Comment:
I can't really see it clearly in Github, but I think you have inconsistent indenting in your code (e.g. tabs vs space). Our C++ codebase standardizes on tabs. You should install EditorConfig.org if you haven't already.